modules/storage-api/src/test/java/org/apache/ignite/internal/storage/AbstractMvTableStorageTest.java

```diff
@@ -118,14 +118,18 @@ void testPartitionIndependence() throws Exception {
 
         UUID txId = UUID.randomUUID();
 
-        RowId rowId0 = partitionStorage0.runConsistently(() -> partitionStorage0.insert(testData0, txId));
+        RowId rowId0 = new RowId(1);
+
+        partitionStorage0.runConsistently(() -> partitionStorage0.addWrite(rowId0, testData0, txId));
 
         assertThat(unwrap(partitionStorage0.read(rowId0, txId)), is(equalTo(unwrap(testData0))));
         assertThat(partitionStorage1.read(rowId0, txId), is(nullValue()));
 
         var testData1 = binaryRow(new TestKey(2, "2"), new TestValue(20, "20"));
 
-        RowId rowId1 = partitionStorage1.runConsistently(() -> partitionStorage1.insert(testData1, txId));
+        RowId rowId1 = new RowId(1);
```

Comment:
paririonId is actually 1 << 8 here. Please add variable storage1PartitionId to use both getOrCreateMvPartition() and rowId1.

Reply:
Done.